src/components/registration/steps.jsx

```diff
@@ -280,6 +282,17 @@ module.exports = {
         },
         render: function () {
             var formatMessage = this.props.intl.formatMessage;
+            var countryOptions = countryData.countryOptions.concat({
+                label: formatMessage({id: 'teacherRegistration.selectCountry'}),
+                disabled: true,
+                selected: true
+            }).sort(function (a, b) {
+                if (a.disabled) return -1;
+                if (b.disabled) return 1;
+                if (a.value === this.props.defaultCountry) return -1;
+                if (b.value === this.props.defaultCountry) return 1;
+                return 0;
+            }.bind(this));
```

Comment:
I tried that – the issue I was running into was the need for `formatMessage`, which we inject at the class level. I was having trouble finding a way to abstract that away, do you have any thoughts on it?